deepcopy_test.go

```diff
@@ -27,41 +27,35 @@ func ExampleAnything() {
 
 	for _, expected := range tests {
 		actual := MustAnything(expected)
-		fmt.Println(actual)
-	}
-	// Output:
-	// "Now cut that out!"
-	// 39
-	// true
-	// false
-	// 2.14
-	// [Phil Harris Rochester van Jones Mary Livingstone Dennis Day]
-	// [Jell-O Grape-Nuts]
+		if !DeepEqual(expected, actual) {
```

Comment:
Changing this would make the example not work anymore, so we can drop this suggestion.